maps/CentralTown/loader.s

```diff
@@ -97,10 +105,10 @@ CentralTown_SpawnMapObjectsForMap:
 	.pool // off_804E734
 off_804E738:
 	// <endpool>
-	.word byte_804E74C
-	.word byte_804E87C
-	.word dword_804E8D0
-	.word dword_804E8D4
-	.word byte_804E8D8
+	.word CentralTown_Map0_ObjectSpawns
+	.word CentralTown_Map1_ObjectSpawns
+	.word CentralTown_Map2_ObjectSpawns
+	.word CentralTown_Map3_ObjectSpawns
+	.word CentralTown_Map4_ObjectSpawns
```

Comment:
I've already decided the styling for map label names. Just prefix the label with the map number name instead of stylizing it like `<MapGroup>_<MapNumber>_<Suffix>`. For reference, the map number names are here: https://gist.github.com/luckytyphlosion/4aaaff3200b7d0de4a79a3cecd2c8899